### ISSUE TITLE
New release required for Symbol overrides change

### ISSUE THREAD

**Issue by dmeehan1968** [darknoon/sketchapp-json-flow-types#11]
Referencing this issue in react-sketchapp - https://github.com/airbnb/react-sketchapp/issues/271

**Comment by jaridmargolin** [comment]
Hi @darknoon.

Thank you for all the work you have already done. Is there anything the community can do to assist in getting a new release published? It would be great to get this out the door to unblock a few tasks in projects that consume this library.

**Comment by mathieudutour** [comment]
done -> `0.3.0`
